<?xml version="1.0"?>
<launch>
     <arg name="input_cloud" default="rslidar_points"/>
    

    <node name="pointcloud_map" pkg="auto_nav" type="pointcloud_map_node" output="screen">
        <param name="input_cloud" value="$(arg input_cloud)"/>
        <rosparam command="load" file="$(find auto_nav)/params/cloud_map_params.yaml"/>

    </node>
    

</launch>